admin/pre-reg/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^$', views.prereg, name='prereg'),
```

Comment:
`drafts/<pk>/update` should be unnecessary. How about implementing `PUT` and/or `PATCH` for `/drafts/<pk>/`?

Also, `/drafts/get` can just be `/drafts/`.